packages/firestore/src/local/local_store.ts

```diff
@@ -347,6 +357,7 @@ class LocalStoreImpl implements LocalStore {
       this.mutationQueue,
       this.persistence.getIndexManager()
     );
+    this.bundleCache = persistence.getBundleCache();
```

Comment:
I think we have done enough iterations on this PR, but ideally this would not be a field on LocalStore (as it breaks tree-shaking). Could `BundleCache` take LocalStore as an argument? Please explore in a follow up.

Reply:
I see. Will do a followup PR.